misc/docker-ci.mk

```diff
@@ -7,17 +7,17 @@ DOCKER_RUN_OPTS=--privileged \
 	-it
 
 ALL:
-	docker run $(DOCKER_RUN_OPTS) $(CONTAINER_NAME) make -f $(CI_MK) _check
+	docker run $(DOCKER_RUN_OPTS) $(CONTAINER_NAME) make -f $(CI_MK) _check CMAKE_ARGS='$(CMAKE_ARGS)' CHECK_ENVS='$(CHECK_ENVS)'
```

Comment:
We have the tradition of declaring defaults for the variables being used in this makefile. Following that tradition,, maybe add `CHECK_ENVS=` at the beginning?